ui/app/pages/home/home.component.js

```diff
@@ -49,6 +50,7 @@ export default class Home extends PureComponent {
     threeBoxLastUpdated: PropTypes.number,
     hasDaiV1Token: PropTypes.bool,
     firstPermissionsRequestId: PropTypes.string,
+    totalUnapprovedCount: PropTypes.number.isRequired,
```

Comment:
It looks like this is the equivalent of `unconfirmedTransactionsCount`, so maybe we don't need to add this?

Reply:
I refactored it to include all of the missing stuff in fa50a00 and 9192b30